diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/classes/CompactInitialization.kt

```diff
@@ -103,6 +106,12 @@ class CompactInitialization(private val configRules: List<RulesConfig>) : Rule("
                             it.treeParent.removeChild(it)
                         }
                     // strip receiver name and move assignment itself into `apply`
+                    if (bodyExpression.treeParent.elementType == FUNCTION_LITERAL
+                            && bodyExpression.firstChildNode.elementType == WHITE_SPACE) {
```

Comment:
Is it error by `KotlinParser` or is something wrong with `getOrCreateApplyBlock`? Where does this incorrectly placed `WHITE_SPACE` node come from? Either way, if you keep this magical hack here, provide some comments for it.